### PR TITLE
add processing sugar and chocolate and fix meatball

### PR DESCRIPTION
**Pull request by Huffie56**
## About The Pull Request
change output of meat to raw meatball instead of cooked meatball

## Why It's Good For The Game
improve recipe consistency

## Changelog

:cl:
add: add the ability to process cocoapod into chocolate without removing standard recipe for it.
code: change output of meat to raw meatball instead of cooked meatball.
/:cl:

### CODE REVIEW

File: code/game/machinery/kitchen/processor.dm [hunk 1]
```diff
@@ -47,6 +47,14 @@
 		input = /obj/item/reagent_container/food/snacks/flour
 		output = /obj/item/reagent_container/food/snacks/spagetti
 
+	spaghetti
```
Comment (Zonespace27) [inline code/game/machinery/kitchen/processor.dm:50]: Hate to tell you but this ain't spaghetti

File: code/game/machinery/kitchen/processor.dm [hunk 2]
```diff
@@ -47,6 +47,14 @@
...
+		input = /obj/item/reagent_container/food/snacks/grown/sugarcane
+		output = /obj/item/reagent_container/food/condiment/sugar
+		
+	spaghetti
```
Comment (Zonespace27) [inline code/game/machinery/kitchen/processor.dm:54]: Ditto

Comment (Huffie56) [reply code/game/machinery/kitchen/processor.dm]: corrected

File: code/game/machinery/kitchen/processor.dm (hunk 1 above)
Comment (Huffie56) [reply code/game/machinery/kitchen/processor.dm]: corrected

**Comment by nauticall** [comment]
please make the title more specific

**Review by ThePiachu — APPROVED** [review]

**Comment by ThePiachu** [comment]
Please also update the wiki pages with the new recipes

**Comment by Huffie56** [comment]
> Please also update the wiki pages with the new recipes

will be done. thanks

**Pull request merged**